src/utils/slug.test.js

```diff
@@ -0,0 +1,23 @@
+const { makeSlug } = require("./slug.js")
+
+describe(`makeSlug()`, () => {
```

Comment:
Check out the configs in `enrich` to stop eslint/pretteir from complaining about this.